scripts/build.spec.ts

```diff
@@ -78,9 +78,16 @@ function fixTypes() {
       let content = readFileSync(dtsFile).toString()
 
       content = content.replace(/^export declare/gm, 'declare')
-
       content = content.replace(/^export \{([\s\n\r]*)\}/gm, '')
 
+      /* Lets explain this amazing regex 🧙‍♂️
```

Comment:
Why this ?
Well, we try with @pbosio to add namespaces, but the rollout types generates a namespace with export fields.
Ok, and what's wrong with this?
We have a regex that checks that the types generated doesn't contain any export word.
```ts
if (content.match(/\bexport\b/)) {
  throw new Error(`The file ${dtsFile} contains exports`)
}
```

### types before regex 
```ts
declare namespace Scene {
  */
  * @ public
  * /
  export type OnEnterLeaveOptions = {
    ...
  }

}
```